force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -81,15 +87,18 @@ public with sharing class ClassesObjectsMethods {
      */
     public static String removeDuplicateWords(String sentence) {
         // Split the sentence into words
-
+        List<String> words = sentence.split(' ');
         // Create a set to store the unique words
-
+        Set<String> uniqueWords = new Set<String>();
         // Add each word to the set
-
+        uniqueWords.addAll(words);
         // Join the unique words back into a sentence
-
+        // Variant 1: works for ApiVersion 58+
+        String newSentence = String.join(uniqueWords, ' ');
+        // Variant 2: works for previous ApiVersions
+        //String newSentence = String.join(new List<String>(uniqueWords), ' ');
         // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        return newSentence;
```

Comment:
really good job. I like to add the list directly to the Set. About the only difference. 

```
 public static String removeDuplicateWords(String sentence) {
        Set<String> uniqueWords = new Set<String>(sentence.split(' '));
        return String.join(uniqueWords, ' ');
    }
    ```